docs/reference/api/ligand_network.rst

```diff
@@ -90,32 +91,6 @@ Atom Map Scorers
 
 Scoring functions for a mapping between ligands. These are used as objective functions for :any:`Ligand Network Planners`.
 
-LOMAP Scorers
```

Comment:
this is all causing warnings for now, so I'll remove and raise an issue about adding this documentation back